From mboxrd@z Thu Jan  1 00:00:00 1970
Return-Path: <linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
Received: from bombadil.infradead.org (bombadil.infradead.org [198.137.202.133])
	(using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
	(No client certificate requested)
	by smtp.lore.kernel.org (Postfix) with ESMTPS id 99EDFD30CC1
	for <linux-rockchip@archiver.kernel.org>; Tue, 13 Jan 2026 20:00:01 +0000 (UTC)
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=bombadil.20210309; h=Sender:
	Content-Transfer-Encoding:Content-Type:List-Subscribe:List-Help:List-Post:
	List-Archive:List-Unsubscribe:List-Id:MIME-Version:References:In-Reply-To:
	Message-ID:Date:Subject:Cc:To:From:Reply-To:Content-ID:Content-Description:
	Resent-Date:Resent-From:Resent-Sender:Resent-To:Resent-Cc:Resent-Message-ID:
	List-Owner; bh=YD76HEq5ybcX8167lhGSDYka+v9YZ0DcRYGUQBlRH2M=; b=YT/tCdsCkhdI8Z
	ND7eXiAK3KIujF/9IGIi7odiLodwrLiBaSLcZ1ho2yzuMSq6x9nx775VigkGVc8WcJszWP0FUYjnx
	BprU4vi+dXh8NoUFRcXeskK3ndn8iFCao8aGDkIImPNaXiZRye9RxvbmS1IxItbsqHLukM7uDXftS
	PJWkFZy1uXodVavXgVFAiFye9Y8tEYUO2XpjEXj1mwPp/8uqsCXgWO5vuAZzhc9USux0//CjbOCh5
	JH92kEKUaPuhu6wCbhJ4kklDq8L20P6JdSSsucDacWupgu3F6EBTwjfWL9939nY2xC5ftk8eVuBx3
	MvsqvfHrp17UaUKsbQgg==;
Received: from localhost ([::1] helo=bombadil.infradead.org)
	by bombadil.infradead.org with esmtp (Exim 4.98.2 #2 (Red Hat Linux))
	id 1vfkYV-00000007eah-2Y5X;
	Tue, 13 Jan 2026 19:59:55 +0000
Received: from mail-ot1-x32a.google.com ([2607:f8b0:4864:20::32a])
	by bombadil.infradead.org with esmtps (Exim 4.98.2 #2 (Red Hat Linux))
	id 1vfkYS-00000007eYZ-1Z3d
	for linux-rockchip@lists.infradead.org;
	Tue, 13 Jan 2026 19:59:53 +0000
Received: by mail-ot1-x32a.google.com with SMTP id 46e09a7af769-7c78d30649aso5552269a34.2
        for <linux-rockchip@lists.infradead.org>; Tue, 13 Jan 2026 11:59:52 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=gmail.com; s=20230601; t=1768334391; x=1768939191; darn=lists.infradead.org;
        h=content-transfer-encoding:mime-version:references:in-reply-to
         :message-id:date:subject:cc:to:from:from:to:cc:subject:date
         :message-id:reply-to;
        bh=c7BnGb+oB3kbI4HRBgeZdx4kG5LrqKQKlKTO5o5vGDY=;
        b=TT9m+qRJJedo6kB/2B6io/I+CJWdK41TlLS6PPE5Z4F635flzT+ZogQxO7S4DH335B
         IIX28+pjgIIUPR8bBvSSMjZzFrnXidfjbvU3V3Rwx5RVvxZSjO/IK+84Rpy2jgosjyMq
         TkOb1H5PKe273xw4pHr/2HUYVn6VoZzmogtoibZbZUhNGaBcFqjXWvCMJtrOcpTueY0b
         yUX2K7Uh+6pMmv6lzo333/4yn/MjuajsHzSQt3Dq+fx/jnYILN8FiJamOOLD0qT8L6h6
         dUB/CZeA3rCstvxsRCicNlaa8zEkJsO/NMghafRxucbflrAmqSR1g0dOAxSI0wUpl3HE
         wPQg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20230601; t=1768334391; x=1768939191;
        h=content-transfer-encoding:mime-version:references:in-reply-to
         :message-id:date:subject:cc:to:from:x-gm-gg:x-gm-message-state:from
         :to:cc:subject:date:message-id:reply-to;
        bh=c7BnGb+oB3kbI4HRBgeZdx4kG5LrqKQKlKTO5o5vGDY=;
        b=sccfu4XEe4hH9lcEMZZVVbCduG9fJuPIEV3XtFrE5qrvCtCD4Uw6UwMvKokxBItZ8B
         DxCZdj3vLiL7+7VwSWmenFQo8oqDPbXtUz4UXNiDo9axTK5HldFoVO8z25GF+Mtx4jwq
         /ir8CCmm03jg/6cei1m7dT8lcewG0UvV/YhkvZSugmby0bg8Zvm1oVbQ0jmQkew7q0OE
         8pvAF9rou2IsOZSrdOP0jOHIyqiLSPMm3SxPV9xoJBcfAncze46qAaHBK8aAnixY3kw+
         FAb8QOjj7RSfPj5MBVeljlz6JWG7Iplr4a4uioZQRFnMUyQw1ojcuNWMkKYn2tam7wSe
         1mhw==
X-Gm-Message-State: AOJu0YzA8Wwd3wxWC0kjNJ9A4D7BXAMllC2zso8tC45fyCPpg5X/FJJU
	Wb8+iAwzZm50wOXuMzrlp/SsHVbNUiRPFGX2czzL1vTUKuflzE6HylwgSn9b+A==
X-Gm-Gg: AY/fxX4v0mS9Szvjnb9goDMSkZN99LG2HhKZPOn1Hi+Wa+ubOOE82P7D2sNx5IDxTuh
	mJkaIRMm3ZEq2CzWRPV54vv5pi/1mkC4HqCw1ssF7jsaKxqUG+CAaLz0Zjhq9xEOU3q3aEGieSm
	KXeb5NT0tmwTDg38oXL39Up5pqSyxsJnBZCTtwSkPWw6PUIvP3EJxiLtRNgTww+GAhbhiHMKy/d
	ObfDCLy68vloTD/71iXurGGpIZFkW40vU9ZcIL9BTiQmcL5u6BMoIav8AvZ5M3qzVChgQd1E1hZ
	nEYQnr71QV/W07wxuqEXfjNEi5QOff4WwAnHwU5gT45S7u5x9N/ijAmTLDAhH7yhGKRiy265APR
	3M78EXfUOpQoORBd8Wxq2GIKYRB9LJlQcK0QDttRuxqVL9E5zoSntnnV/DCmCfiBo4XIVEwSdyH
	4K8EmYYpE9
X-Received: by 2002:a05:6830:4119:b0:7c6:a2da:ce4b with SMTP id 46e09a7af769-7cfc8a31752mr227469a34.5.1768334391060;
        Tue, 13 Jan 2026 11:59:51 -0800 (PST)
Received: from localhost.localdomain ([2600:1700:fb0:1bc0::54])
        by smtp.gmail.com with ESMTPSA id 46e09a7af769-7ce47801d4dsm16221119a34.5.2026.01.13.11.59.50
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Tue, 13 Jan 2026 11:59:50 -0800 (PST)
From: Chris Morgan <macroalpha82@gmail.com>
To: linux-rockchip@lists.infradead.org
Cc: dri-devel@lists.freedesktop.org,
	devicetree@vger.kernel.org,
	dmitry.torokhov@gmail.com,
	simona@ffwll.ch,
	airlied@gmail.com,
	tzimmermann@suse.de,
	mripard@kernel.org,
	maarten.lankhorst@linux.intel.com,
	jesszhan0024@gmail.com,
	neil.armstrong@linaro.org,
	jagan@edgeble.ai,
	heiko@sntech.de,
	conor+dt@kernel.org,
	krzk+dt@kernel.org,
	robh@kernel.org,
	aweinzerl13@yahoo.com,
	Chris Morgan <macromorgan@hotmail.com>
Subject: [PATCH 4/6] dt-bindings: arm: rockchip: Add Anbernic RG-DS
Date: Tue, 13 Jan 2026 13:57:19 -0600
Message-ID: <20260113195721.151205-5-macroalpha82@gmail.com>
X-Mailer: git-send-email 2.43.0
In-Reply-To: <20260113195721.151205-1-macroalpha82@gmail.com>
References: <20260113195721.151205-1-macroalpha82@gmail.com>
MIME-Version: 1.0
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20260113_115952_411753_BEBC441A 
X-CRM114-Status: UNSURE (   8.29  )
X-CRM114-Notice: Please train this message.
X-BeenThere: linux-rockchip@lists.infradead.org
X-Mailman-Version: 2.1.34
Precedence: list
List-Id: Upstream kernel work for Rockchip platforms <linux-rockchip.lists.infradead.org>
List-Unsubscribe: <http://lists.infradead.org/mailman/options/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=unsubscribe>
List-Archive: <http://lists.infradead.org/pipermail/linux-rockchip/>
List-Post: <mailto:linux-rockchip@lists.infradead.org>
List-Help: <mailto:linux-rockchip-request@lists.infradead.org?subject=help>
List-Subscribe: <http://lists.infradead.org/mailman/listinfo/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=subscribe>
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
Sender: "Linux-rockchip" <linux-rockchip-bounces@lists.infradead.org>
Errors-To: linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org

From: Chris Morgan <macromorgan@hotmail.com>

Add compatible string for the Anbernic RG-DS.

Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
---
 Documentation/devicetree/bindings/arm/rockchip.yaml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/rockchip.yaml b/Documentation/devicetree/bindings/arm/rockchip.yaml
index d496421dbd87..4398f576e47f 100644
--- a/Documentation/devicetree/bindings/arm/rockchip.yaml
+++ b/Documentation/devicetree/bindings/arm/rockchip.yaml
@@ -60,6 +60,12 @@ properties:
               - anbernic,rg-arc-s
           - const: rockchip,rk3566
 
+      - description: Anbernic RK3568 Handheld Gaming Console
+        items:
+          - enum:
+              - anbernic,rg-ds
+          - const: rockchip,rk3568
+
       - description: Ariaboard Photonicat
         items:
           - const: ariaboard,photonicat
-- 
2.43.0


_______________________________________________
Linux-rockchip mailing list
Linux-rockchip@lists.infradead.org
http://lists.infradead.org/mailman/listinfo/linux-rockchip

